contrib/lang/scala/packages.el

```diff
@@ -47,6 +47,78 @@ which require an initialization must be listed explicitly in the list.")
         (kbd "n") 'forward-button
         (kbd "N") 'backward-button)
 
+      (defun ensime-gen-and-reload()
```

Comment:
I've added a new function which generates an .ensime and reloads ensime, which is useful when the project changes. Sometimes it's also necessary to `rm -r .ensime_cache` in the project root to force ensime to notice changes to e.g. dependencies.